### PR TITLE
Garmin Foretrex 401

### PR DESCRIPTION
**Pull request by kavinsky**
**Work in Progress**

some screenshots of early device dialog:

* Track page, level 1: http://i.imgur.com/jAY21VU.jpg
* Track page, level 2: http://i.imgur.com/g0Tvjvg.jpg
* Track page, level 3: http://i.imgur.com/7F9Iqwx.jpg
* Elevation page: http://i.imgur.com/EhUTq1i.jpg
* Timer page: http://i.imgur.com/4JooUWK.jpg

The font you see is a placeholder. Im planning to replace with another that simulates a dot matrix screen.

Menu workflow (sorry, in spanish): http://i.imgur.com/7UQyF6H.png

_I don't actualy own one so I may make some mistakes along the way. If you happen to be lucky enough to have one yourself I'd appreciate any feedback I can get._

### CODE REVIEW

**Comment by gienkov** [comment]
:+1: I always like new toys to play with ^^

File: addons/foretrex401/functions/fnc_empty.sqf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#include "script_component.hpp"
+
+diag_log text format["This is here as an example!!!"];
```
Comment (SilentSpike) [inline addons/foretrex401/functions/fnc_empty.sqf]: This file can be removed

File: addons/foretrex401/XEH_preInit.sqf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+PREP(empty);
```
Comment (SilentSpike) [inline addons/foretrex401/XEH_preInit.sqf]: This line can be removed

**Comment by bux578** [comment]
@kavinsky   Status?

**Comment by kavinsky** [comment]
@bux578 paused, ill retake the work on september.

**Comment by Glowbal** [comment]
Closing, no activity.

**Pull request closed without merging**